src/url_aggregator.cpp

```diff
@@ -426,6 +426,7 @@ bool url_aggregator::set_host_or_hostname(const std::string_view input) {
     // Let host be the result of host parsing host_view with url is not special.
     if (host_view.empty()) {
       clear_base_hostname();
+      add_authority_slashes_if_needed();
```

Comment:
should we just move this line to `clear_base_hostname()` function?

Reply:
Let us try that.